pode.build.ps1

```diff
@@ -88,12 +87,13 @@ function Install-PodeBuildModule($name) {
 }
 
 function Invoke-PodeBuildDotnetBuild($target) {
-    dotnet build --configuration Release --self-contained --framework $target
+    $AssemblyVersion=($Version)? "-p:Version=$Version" : ''
+    dotnet build --configuration Release --self-contained --framework $target $AssemblyVersion
     if (!$?) {
         throw "dotnet build failed for $($target)"
     }
 
-    dotnet publish --configuration Release --self-contained --framework $target --output ../Libs/$target
+    dotnet publish --configuration Release --self-contained --framework $target $AssemblyVersion --output ../Libs/$target
```

Comment:
Create a separate bug report for this, and a separate PR. Don't do it as a part of the OpenAPI work. It's also got the PS7 ternary issue 😜

Reply:
done